packages/prisma/schema.prisma

```diff
@@ -10,13 +10,13 @@ datasource db {
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["views"]
+  output          = "./client"
 }
 
 generator zod {
-  provider      = "zod-prisma"
-  output        = "./zod"
-  imports       = "./zod-utils"
-  relationModel = "default"
+  provider         = "zod-prisma-types"
```

Comment:
@emrysal you're so brave for this

Reply:
I kinda get the move since "zod-prisma" it's been long abandoned. Blocking the upgrade. Kinda makes sense to do it now.

Reply:
It was actually blocking the upgrade as the generated types did an import to the "old prisma" files that could not be fixed without forking; which I opted against.